subsys/bluetooth/host/gatt.c

```diff
@@ -1307,7 +1308,11 @@ void bt_gatt_init(void)
 	/* Submit work to Generate initial hash as there could be static
 	 * services already in the database.
 	 */
+#if defined(CONFIG_BT_LONG_WQ)
+	k_work_schedule_for_queue(bt_long_wq_get(), &db_hash.work, DB_HASH_TIMEOUT);
+#else
 	k_work_schedule(&db_hash.work, DB_HASH_TIMEOUT);
+#endif /* CONFIG_BT_LONG_WQ */
```

Comment:
Can't we use `IS_ENABLED` now that we use `bt_long_wq_get()`?

Reply:
yeah, sorry I updated the PR a few minutes after with the selection done in `long_work.h`. I'll update it again when we decide on https://github.com/zephyrproject-rtos/zephyr/pull/46131#discussion_r887797468